apps/frontend/src/hooks/useTimeElapsed.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+import { DateTime } from 'luxon';
+
+const UseTimeElapsed = (createdAt) => {
```

Comment:
```suggestion
const useTimeElapsed = (createdAt) => {
```

A hook name start with a lower case.

Reply:
Yes